Please build and publish a Wheel file on Python Package Index for Linux on aarch64

Tried to build and run an Python application on Ubuntu 20.04 LTS with aarch64, and this application depends on [highdicom](https://pypi.org/project/highdicom/), which in turn depend on pillow-jpls.

Importing highdicom failed due to the failure on building the pillow-jpls Wheel.

We can look into the Wheel build failure, output enclosed below points to the profile config, but it would be great that a package for Linux on aarch64 is already in the Python Package Index.

Thank you so much.

```
$ pip install highdicom
Defaulting to user installation because normal site-packages is not writeable
Collecting highdicom
  Using cached highdicom-0.22.0-py3-none-any.whl.metadata (3.8 kB)
Requirement already satisfied: pydicom!=2.4.0,>=2.3.0 in /home/nvidia/.local/lib/python3.8/site-packages (from highdicom) (2.4.3)
Requirement already satisfied: numpy>=1.19 in /home/nvidia/.local/lib/python3.8/site-packages (from highdicom) (1.24.4)
Requirement already satisfied: pillow>=8.3 in /home/nvidia/.local/lib/python3.8/site-packages (from highdicom) (10.1.0)
Collecting pillow-jpls>=1.0 (from highdicom)
  Using cached pillow-jpls-1.2.0.tar.gz (18 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Using cached highdicom-0.22.0-py3-none-any.whl (825 kB)
Building wheels for collected packages: pillow-jpls
  Building wheel for pillow-jpls (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pillow-jpls (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [95 lines of output]
      
      
      --------------------------------------------------------------------------------
      -- Trying 'Ninja' generator
      --------------------------------
      ---------------------------
      ----------------------
      -----------------
      ------------
      -------
      --
      CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
        Compatibility with CMake < 3.5 will be removed from a future version of
        CMake.
      
        Update the VERSION argument <min> value or use a ...<max> suffix to tell
        CMake that the project does not need compatibility with older versions.
      
      Not searching for unused variables given on the command line.
      
      -- The C compiler identification is GNU 9.4.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /usr/bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- The CXX compiler identification is GNU 9.4.0
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Configuring done (0.6s)
      -- Generating done (0.0s)
      -- Build files have been written to: /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5/_cmake_test_compile/build
      --
      -------
      ------------
      -----------------
      ----------------------
      ---------------------------
      --------------------------------
      -- Trying 'Ninja' generator - success
      --------------------------------------------------------------------------------
      
      Configuring Project
        Working directory:
          /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5/_skbuild/linux-aarch64-3.8/cmake-build
        Command:
          /tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/cmake/data/bin/cmake /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5 -G Ninja -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/ninja/data/bin/ninja --no-warn-unused-cli -DCMAKE_INSTALL_PREFIX:PATH=/tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5/_skbuild/linux-aarch64-3.8/cmake-install/pillow_jpls -DPYTHON_VERSION_STRING:STRING=3.8.10 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/skbuild/resources/cmake -DPYTHON_EXECUTABLE:PATH=/usr/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPYTHON_LIBRARY:PATH=/usr/lib/aarch64-linux-gnu/libpython3.8.so -DPython_EXECUTABLE:PATH=/usr/bin/python3 -DPython_ROOT_DIR:PATH=/usr -DPython_FIND_REGISTRY:STRING=NEVER -DPython_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython3_EXECUTABLE:PATH=/usr/bin/python3 -DPython3_ROOT_DIR:PATH=/usr -DPython3_FIND_REGISTRY:STRING=NEVER -DPython3_INCLUDE_DIR:PATH=/usr/include/python3.8 -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/ninja/data/bin/ninja -DCMAKE_BUILD_TYPE:STRING=Release
      
      Not searching for unused variables given on the command line.
      -- The C compiler identification is GNU 9.4.0
      -- The CXX compiler identification is GNU 9.4.0
      -- Detecting C compiler ABI info
      -- Detecting C compiler ABI info - done
      -- Check for working C compiler: /usr/bin/cc - skipped
      -- Detecting C compile features
      -- Detecting C compile features - done
      -- Detecting CXX compiler ABI info
      -- Detecting CXX compiler ABI info - done
      -- Check for working CXX compiler: /usr/bin/c++ - skipped
      -- Detecting CXX compile features
      -- Detecting CXX compile features - done
      -- Downloading conan.cmake from https://github.com/conan-io/cmake-conan
      -- Conan: checking conan executable
      -- Conan: Found program /tmp/pip-build-env-stdikedt/overlay/bin/conan
      -- Conan: Version found Conan version 2.0.14
      
      -- Conan executing: /tmp/pip-build-env-stdikedt/overlay/bin/conan install /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5 --build missing --settings build_type=Release --settings compiler=gcc --settings compiler.version=9 --settings compiler.libcxx=libstdc++11
      ERROR: The default build profile '/home/nvidia/.conan2/profiles/default' doesn't exist.
      You need to create a default profile (type 'conan profile detect' command)
      or specify your own profile with '--profile:build=<myprofile>'
      CMake Error at _skbuild/linux-aarch64-3.8/cmake-build/conan.cmake:631 (message):
        Conan install failed='1'
      Call Stack (most recent call first):
        CMakeLists.txt:17 (conan_cmake_install)
      
      
      -- Configuring incomplete, errors occurred!
      Traceback (most recent call last):
        File "/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/skbuild/setuptools_wrap.py", line 666, in setup
          env = cmkr.configure(
        File "/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/skbuild/cmaker.py", line 357, in configure
          raise SKBuildError(msg)
      
      An error occurred while configuring with CMake.
        Command:
          /tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/cmake/data/bin/cmake /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5 -G Ninja -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/ninja/data/bin/ninja --no-warn-unused-cli -DCMAKE_INSTALL_PREFIX:PATH=/tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5/_skbuild/linux-aarch64-3.8/cmake-install/pillow_jpls -DPYTHON_VERSION_STRING:STRING=3.8.10 -DSKBUILD:INTERNAL=TRUE -DCMAKE_MODULE_PATH:PATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/skbuild/resources/cmake -DPYTHON_EXECUTABLE:PATH=/usr/bin/python3 -DPYTHON_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPYTHON_LIBRARY:PATH=/usr/lib/aarch64-linux-gnu/libpython3.8.so -DPython_EXECUTABLE:PATH=/usr/bin/python3 -DPython_ROOT_DIR:PATH=/usr -DPython_FIND_REGISTRY:STRING=NEVER -DPython_INCLUDE_DIR:PATH=/usr/include/python3.8 -DPython3_EXECUTABLE:PATH=/usr/bin/python3 -DPython3_ROOT_DIR:PATH=/usr -DPython3_FIND_REGISTRY:STRING=NEVER -DPython3_INCLUDE_DIR:PATH=/usr/include/python3.8 -DCMAKE_MAKE_PROGRAM:FILEPATH=/tmp/pip-build-env-stdikedt/overlay/lib/python3.8/site-packages/ninja/data/bin/ninja -DCMAKE_BUILD_TYPE:STRING=Release
        Source directory:
          /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5
        Working directory:
          /tmp/pip-install-by1vi5p2/pillow-jpls_8a02b44b4c8b44388bbb25fbc446c1c5/_skbuild/linux-aarch64-3.8/cmake-build
      Please see CMake's output for more information.
      
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pillow-jpls
Failed to build pillow-jpls
ERROR: Could not build wheels for pillow-jpls, which is required to install pyproject.toml-based projects
```

--- a/.github/workflows/build_deploy.yml
+++ b/.github/workflows/build_deploy.yml
@@ -36,12 +36,16 @@ jobs:
       matrix:
         os: [ ubuntu-22.04, windows-2022, macos-12 ]
         pyversion: [ cp38, cp39, cp310, cp311, cp312 ]
-        arch: [ auto32, auto64, arm64 ]
+        arch: [ auto32, auto64, aarch64, arm64 ]
         exclude:
           - os: macos-12
             arch: auto32
+          - os: macos-12
+            arch: aarch64
           - os: windows-2022
             arch: arm64
+          - os: windows-2022
+            arch: aarch64
           - os: ubuntu-22.04
             arch: arm64
           - pyversion: cp312
@@ -58,6 +62,12 @@ jobs:
         uses: ilammy/msvc-dev-cmd@v1
         with:
           arch: ${{ matrix.arch == 'auto64' && 'amd64' || 'amd64_x86' }}
+
+      - name: Install QEMU for Arm builds
+        if: matrix.arch == 'aarch64'
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: arm64
 
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.16.2

